tests/Tests/100__deploy/100__installation/108__jupyterhub.robot

```diff
@@ -38,11 +38,7 @@ Verify PostgreSQL Is Not Deployed When AWS RDS Is Enabled
     [Tags]    Sanity
     ...       Tier1
     ...       ODS-336
-    ${cluster_platform_type}=    Fetch Cluster Platform Type
-    Skip if    "${cluster_platform_type}" != "AWS"
-    ...    This test only applies to AWS clusters
-    Run Keyword And Expect Error    ResourceOperationFailed: Get failed\nReason: Not Found
-    ...    Fetch JupyterHub DB Pods Info
+    Skip      msg=PostgreSQL Pod is removed after KFNBC migration
```

Comment:
should we check the postgres sql pod is actually gone? @pablofelix @tarukumar

Reply:
I don't think it's needed. this changes are there because scrips were failing due to missing postgres. We can do that but not for this PR